### PR TITLE
Bump kubekins to v20170927-54229024

### PR DESCRIPTION
**Pull request by krzyzacy**
includes https://github.com/kubernetes/test-infra/pull/4777

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 ORIG_CWD = os.getcwd()  # Checkout changes cwd
 
 # Note: This variable is managed by experiment/bump_e2e_image.sh.
```
Comment (BenTheElder) [inline scenarios/kubernetes_e2e.py]: not so experimental anymore, is it? :-)

Comment (krzyzacy) [reply scenarios/kubernetes_e2e.py]: yeah I'll move it out to somewhere. @zmerlynn your feature graduated from alpha to beta now!

**Comment by krzyzacy** [comment]
(This did not touch anything except for the node tests, I'll just merge it to see if it fixes the failing tests)

**Pull request merged**